pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
So if I have a CSC `{ RunAsUser: 1234 }`, that nullifies my PSC `{ Capabilities: {...}; SELinux: {...}; /* etc */ }` ??  That still feels wrong